packages/web/src/components/AddPlayerForm.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
+
+type ParticipantFormData = {
+  name: string;
+  videoUrl: string;
+}
+
+type FormErrors = {
+  name?: string;
+  videoUrl?: string;
+}
+
+const ParticipantForm = () => {
+  const [formData, setFormData] = useState<ParticipantFormData>({
+    name: '',
+    videoUrl: ''
+  });
+
+  const [players, setPlayers] = useState<ParticipantFormData[]>([]);
+  const [errors, setErrors] = useState<FormErrors>({});
+
+  const isValidYouTubeUrl = (url: string): boolean => {
```

Comment:
it seems that isValidYouTubeUrl and getYouTubeVideoId do exactly the same thing except for the return type ... you could simplify by doing:

```ts
const isValidYouTubeUrl = (url: string): boolean => getYouTubeVideoId(url) != null
```